keyboards/handwired/polykybd/lang/_patch_xlsx.py

```
@@ -0,0 +1,93 @@
+            if cellval is None: continue
+            c = base+k
+            ref = f"{colname(c)}{r}"
+            kind,val = cellval
+            xml = num_cell(ref,val) if kind=="num" else str_cell(ref,val)
+            appended.setdefault(r,[]).append((c,xml))
+
+newmaxcol = 2 + (n+len(ORDER))*4 - 1       # last col used (incl LCID at base+3)
+print(f"new max col index = {newmaxcol} ({colname(newmaxcol)})")
+
+# read sheet2.xml
+tmp="/tmp/_xlsx_work"; shutil.rmtree(tmp,ignore_errors=True); os.makedirs(tmp)
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/_xlsx_work"

(S108)